README.md

```diff
@@ -1,49 +1,55 @@
+* Subsamples the metadata across time and geography
+* Downloads fastq files for subsampled metadata from NCBI SRA using [fasterq-dump](https://github.com/ncbi/sra-tools/wiki/HowTo:-fasterq-dump)
+* Assigns lineages and identifies drug resistance variants for each sample using [tb-profiler](https://github.com/jodyphelan/TBProfiler)
+* Creates a multi-sample fasta alignment using [snippy](https://github.com/tseemann/snippy) with low-confidence regions masked following [Marin et al. 2022](https://academic.oup.com/bioinformatics/article/38/7/1781/6502279)
+* Creates a multi-sample VCF of informative sites using a custom script
+* Performs phylogenetic reconstruction using [IQTREE](http://www.iqtree.org/)
 
-This is the [Nextstrain](https://nextstrain.org/) build for tuberculosis, visible at [nextstrain.org/tb](https://nextstrain.org/tb).
+The results of running this workflow are publicly visible at nextstrain.org/tb
 
-The build encompasses preparing data for analysis, doing quality control, performing analyses, and saving the results in a format suitable for visualization (with [auspice][]). 
-This involves running components of Nextstrain such as [augur][] and [auspice][].
 
-All tuberculosis-specific steps and functionality for the Nextstrain pipeline should be housed in this repository.
 
-## Tutorial
+## Installation
+Follow the [standard installation instructions](https://docs.nextstrain.org/en/latest/install.html) for Nextstrain's suite of software tools.
 
-There is a [tutorial](https://nextstrain.org/docs/getting-started/tb-tutorial) on how to run this build.
+## Usage:
 
-## Usage
+> NOTE: Running this workflow will most likely require more compute resources than what is available on your local computer.
 
-If you're unfamiliar with Nextstrain builds, you may want to follow our
-[quickstart guide][] first and then come back here.
+```
+nextstrain build --image ghcr.io/nextstrain/tb-docker-test:16923552818 .
+```
 
-The easiest way to run this pathogen build is using the [Nextstrain
-command-line tool][nextstrain-cli]:
+## Storing tbprofiler and snippy results
+For SRA samples that have already been analyzed in previous runs of this workflow, results of tb-profiler and snippy analyses are stored in the `nextstrain-data` AWS S3 bucket:
+### tb-profiler
+* Results files:
+	* `{sample}.results.json`
+* Path: `nextstrain-data/files/workflows/tb/`
+### snippy 
+* Results files:
+	* `{sample}/snps.aligned.fa`
+	* `{sample}/snps.vcf`
+* Path: `nextstrain-data/files/workflows/snippy`
 
-    nextstrain build .
+These results files should be deleted from the S3 bucket if changes are made to the workflow that would influence the files, such as changes to the parameters used in the tb-profiler or snippy analysis steps, updates to the tb-profiler or snippy installations, or addition of new sequence quality filtering steps prior to running tb-profiler or snippy.
```

Comment:
There's lots of technical improvements we can make to this (storing metadata on the object, encoding metadata in the file path etc) - I'd suggest making an issue about this and maybe someone can take it on as a future project.